source/_components/switch.raspyrfm.markdown

```diff
@@ -0,0 +1,130 @@
+---
+layout: page
+title: "RaspyRFM Switch"
+description: "Instructions on how to integrate RaspyRFM switches into Home Assistant."
+date: 2018-12-08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: seegelsysteme.png
+ha_category: Switch
+ha_release: 0.85
+---
+
+The `raspyrfm` component adds support for cheap RC 433 MHz outlets via one of the supported gateways.
+
+Initially this component was created to support the Simple Solutions `ConnAir` gateway which has been discontinued. There are custom alternatives that reimplemented the protocol used by the ConnAir though like this [ConnAir emulator](https://github.com/Phunkafizer/RaspyRFM/blob/master/connair.py) wich can be used in conjunction with the [RaspyRFM-II](https://www.seegel-systeme.de/produkt/raspyrfm-ii) rc module for a Raspberry Pi.
+
+Other vendors of 433 MHz rc outlets have also created gateways that use a very similar protocol and can also be used with this component like the Intertechno [ITGW-433 LAN Gateway](https://www.intertechno24.de/LAN-Gateway/Gateway-ITGW-433.html) 
+ 
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  platform: raspyrfm
+  gateway_manufacturer: Seegel Systeme
+  gateway_model: RaspyRFM
+  host: 127.0.0.1              # Optional
```

Comment:
The first config example should be minimal and only include required items.

Reply:
Ok